### PR TITLE
Added information about latest versions of Cucumber JVM adapters

### PR DESCRIPTION
**Pull request by letsrokk**
Following PR https://github.com/allure-framework/allure-java/pull/286, adding information about latest version of Cucumber JVM adapters to Docs

+

Added `Configuration` block (`allure-results` and and Issue/TMS link patterns)

### CODE REVIEW

File: docs/integration/cucumberjvm.adoc [hunk 1]
```diff
@@ -2,12 +2,17 @@
 
 == Installation
 
-The available latest version of `allure-cucumber-jvm`:
+Each major version of Cucumber JVM requires particular version of Allure Cucumber JVM adapter.
```
Comment (james-bjss) [inline docs/integration/cucumberjvm.adoc]: Sorry, very small change:
`Each major version of Cucumber JVM requires **a** particular version of Allure Cucumber JVM adapter.`

File: docs/integration/cucumberjvm.adoc [hunk 2]
```diff
@@ -2,12 +2,17 @@
...
+The available latest version of adapters:
+
+- Cucumber JVM 1.x - `allure-cucumber-jvm` image:https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber-jvm.svg[Allure Cucumber JVM 1.x]
+- Cucumber JVM 2.x - `allure-cucumber2-jvm` image:https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber2-jvm.svg[Allure Cucumber JVM 2.x]
+- Cucumber JVM 3.x - `allure-cucumber3-jvm` image:https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber3-jvm.svg[Allure Cucumber JVM 3.x]
+- Cucumber JVM 4.x - `allure-cucumber4-jvm` image:https://img.shields.io/maven-central/v/io.qameta.allure/allure-cucumber4-jvm.svg[Allure Cucumber JVM 4.x]
 
 === Maven
-Simply add `allure-cucumber-jvm` plugin as dependency to your project and add it to CucumberOptions:
+Simply add `allure-cucumber4-jvm` plugin as dependency to your project and add it to CucumberOptions:
```
Comment (james-bjss) [inline docs/integration/cucumberjvm.adoc]: Simply add `allure-cucumber4-jvm` plugin as **a** dependency to your project and add it to CucumberOptions:

File: docs/integration/cucumberjvm.adoc (hunk 1 above)
Comment (letsrokk) [reply docs/integration/cucumberjvm.adoc]: Thanks!

File: docs/integration/cucumberjvm.adoc (hunk 2 above)
Comment (letsrokk) [reply docs/integration/cucumberjvm.adoc]: Thanks!

File: docs/features/configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                    <allure.link.tms.pattern>https://example.org/browse/{}</allure.link.tms.pattern>
+                </systemPropertyVariables>
+            </configuration>
+            ...
+        </plugin>
+    </plugins>
+</build>
+----
+
+Or by placing `allure.properties` file to resources folder: `src/test/resources`
```
Comment (baev) [inline docs/features/configuration.adoc]: I'd like to have `allure.properties` file mentioned first as the main (*recommended*) way to configure Allure

**Comment by baev** [comment]
@letsrokk @james-bjss thanks!

**Pull request merged**